### PR TITLE
Feat navigation icon scroll

### PR DESCRIPTION
**Pull request by Karim94**
#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/1132/mobile-scroll
#### Testing instructions  
N/A
#### Notes
In tabs.js , the onPress function needs to be changed from anonymous arrow function

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: app/components/messaging/chat-list.js [hunk 1]
```diff
@@ -130,7 +131,10 @@ export default class ChatList extends SafeComponent {
                 onEndReachedThreshold={20}
                 onContentSizeChange={this.scroll}
                 enableEmptySections
-                ref={sv => { this.scrollView = sv; }}
+                ref={sv => {
+                    this.scrollView = sv;
+                    uiState.currentScrollView = sv;
+                }}
```
Comment (seavan) [inline app/components/messaging/chat-list.js]: please move to bound function

File: app/components/contacts/contact-list.js [hunk 2]
```diff
@@ -70,6 +71,10 @@ export default class ContactList extends SafeComponent {
                 keyExtractor={item => item.username || item.email}
                 renderItem={this.item}
                 renderSectionHeader={this.header}
+                ref={sv => {
+                    this.scrollView = sv;
+                    uiState.currentScrollView = sv;
+                }}
```
Comment (seavan) [inline app/components/contacts/contact-list.js]: please move to bound function

File: app/components/layout/tabs.js [hunk 3]
```diff
@@ -46,7 +46,26 @@ export default class Tabs extends SafeComponent {
...
+                                    itemIndex: 0,
+                                    sectionIndex: 0,
+                                    viewOffset: vars.contactListHeaderHeight,
+                                    viewPosition: 0
+                                });
+                        }
+                    } else {
+                        routerMain[route]();
+                    }
+                }}
```
Comment (seavan) [inline app/components/layout/tabs.js]: should check currentScrollView for null
should be put into bound function

File: app/components/settings/settings-level-1.js [hunk 4]
```diff
@@ -64,7 +65,12 @@ export default class SettingsLevel1 extends SafeComponent {
             <SettingsItem title="button_upgrade" onPress={() => settingsState.upgrade()} />;
         return (
             <View style={bgStyle}>
-                <ScrollView contentContainerStyle={svStyle}>
+                <ScrollView
+                    contentContainerStyle={svStyle}
+                    ref={sv => {
+                        this.scrollView = sv;
+                        uiState.currentScrollView = sv;
+                    }} >
```
Comment (seavan) [inline app/components/settings/settings-level-1.js]: needs to be put into bound function,

**Comment by ebilg** [comment]
can be merged

**Pull request merged**